dist/clipboard.js

```diff
@@ -26,8 +28,11 @@ if (Element && !Element.prototype.matches) {
  * @return {Function}
  */
 function closest (element, selector) {
-    while (element && element !== document) {
-        if (element.matches(selector)) return element;
+    while (element && element.nodeType !== DOCUMENT_NODE_TYPE) {
+        if (typeof element.matches === 'function' &&
+            element.matches(selector)) {
+          return element;
```

Comment:
We don't actually use this file, but given it's generated I wouldn't worry about it so much. I just did this so it can be kept in sync with the src and lib folders in case we need to compare for some reason.